README.md

```diff
@@ -135,6 +135,35 @@ Sets a timeout for transmitting a request to the proxied server.
 The timeout is set only between two successive write operations, not for the transmission of the whole request.  
 If the proxied server does not receive anything within this time, the connection is closed.
 
+proxy_connect_address
+---------------------------
+
+Syntax: **proxy_connect_address `address [transparent] | off`**
+Default: `none`
+Context: `server`
+
+Specifiy an IP address of the proxied server. The address can contain variables. The special value is equal to none,
```

Comment:
`The special value is equal to none,`

->

`The default value is none,`

Reply:
uh, it's a typo.